README.md

```diff
@@ -766,6 +766,111 @@ The `/api/v1/export` endpoint should return the following response:
 Extra labels may be added to all the imported time series by passing `extra_label=name=value` query args.
 For example, `/api/put?extra_label=foo=bar` would add `{foo="bar"}` label to all the ingested metrics.
 
+## How to send data from NewRelic infrastructure agent
+
+VictoriaMetrics accepts data from NewRelic infrastructure agent API at `/api/v1/newrelic/infra/v2/metrics/events/bulk` path.
+NewRelic's infrastructure agent sends so-called [Events](https://docs.newrelic.com/docs/data-apis/understand-data/new-relic-data-types/#event-data)
+which are transformed to the [Prometheus exposition format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-based-format).
+
+By default, NewRelic's infrastructure agent sends next types of the infrastructure events:
```

Comment:
I believe we should explain `Sending metrics to VictoriaMetrics` first and then talk about data mapping.

Reply:
updated